pyproject.toml

```diff
@@ -2,24 +2,32 @@
 requires = ["poetry-core>=1.1.0a6"]
 build-backend = "poetry.core.masonry.api"
 
+[tool.mypy]
```

Comment:
should this section be added to volttron-core?

Reply:
also, python_version should be >3.8 < 4 not sure what this is actually doing here.  but 3.9 is arbitrary

Reply:
shouldn't these be specified in the pyproject.yml file anyways?

Reply:
> should this section be added to volttron-core?

These were added  after I initially ran the pre-commit hooks on this repo. There was some configuration I wanted on mypy for this specific repo when I was trying to get pre-commit hooks to pass on this repo. If volttron-core wants to configure mypy, I'm all for it.

Reply:
> also, python_version should be >3.8 < 4 not sure what this is actually doing here. but 3.9 is arbitrary

Makes sense. I can update it.

Reply:
> shouldn't these be specified in the pyproject.yml file anyways?

What is pyproject.yml? These settings were originally in a mypy.ini file. But I realized that you could put this in pyproject.toml and just have one centralized configuration file.

Reply:
pyproject.toml file is the poetry file for configuration.

Reply:
You wrote pyproject.**yml** in one of the [earlier comments](https://github.com/eclipse-volttron/volttron-platform-driver/pull/13#discussion_r948517985). I think you meant pyproject.toml. And yes, it should be in pyproject.toml.